docs/resources/kms_key.md

```diff
@@ -49,6 +50,13 @@ The following arguments are supported:
 
 * `tags` - (Optional, Map) Specifies the key/value pairs to associate with the kms key.
 
+* `origin` - (Optional, String, ForceNew) Specifies the source of the kms key. Valid values are **kms** and **external**.
+  The default value is **kms**. Changing this creates a new key.
+
+* `key_usage` - (Optional, String, ForceNew) Specifies the key usage. The default value for a symmetric key
+  is **ENCRYPT_DECRYPT**, the default value for an asymmetric key is **SIGN_VERIFY**.
+  The value can be **ENCRYPT_DECRYPT** or **SIGN_VERIFY**. Changing this creates a new key.
```

Comment:
Actually， **ENCRYPT_DECRYPT** represents the symmetric key and **SIGN_VERIFY** represents the asymmetric key.
they are not the default value of them.